fenix/app/src/test/java/org/mozilla/fenix/home/DefaultSessionControlControllerTest.kt

```diff
@@ -1051,6 +1052,27 @@ class DefaultSessionControlControllerTest {
         }
     }
 
+    @Test
+    fun `WHEN top site long clicked is called THEN report the top site long click telemetry`() {
+        assertNull(TopSites.longPress.testGetValue())
+
+        val topSite = TopSite.Provided(
+            id = 1L,
+            title = "Mozilla",
+            url = "mozilla.org",
+            clickUrl = "",
+            imageUrl = "",
+            impressionUrl = "",
+            createdAt = 0,
+        )
+
+        createController().handleTopSiteLongClicked(topSite)
+
+        val shownEvent = TopSites.longPress.testGetValue()!!
+        assertNotNull(TopSites.longPress.testGetValue())
```

Comment:
will the `!!` unwrapping make `assertNotNull` redundant?

Reply:
Removed the assertion